app/cdap/components/ConnectionList/__test__/ConnectionList.test.tsx

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright © 2022 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+import { render } from '@testing-library/react';
+import * as apiHelpers from 'components/Connections/Browser/SidePanel/apiHelpers';
```

Comment:
import apiHelpers from -- don't need the asterisks

Reply:
asterisks is needed as we are importing the file as a module and using the functions that are exposed for mocking the function